### DISCUSSION TITLE
Erro import QListMapTiledCollectionPoint

### DISCUSSION THREAD

**Discussion by MarcosAndre28** [osmandapp/OsmAnd#14210] [Q&A]
Hello, I have an error compiling the project.
import error when compiling the project
![import erro](https://user-images.githubusercontent.com/34040590/166928311-bb3abcf4-89d9-4bde-be97-546f11e7bb48.png)
![image](https://user-images.githubusercontent.com/34040590/166930772-08bf84be-42d8-4581-ac79-dcaa4ddd1abb.png)
I tried to solve it in several ways but I was not successful.

**Comment by vshcherb** [comment]
I think it comes from newer version of osmand ndk library, please clean up gradle or call gradle with refresh dependencies.

**Reply by MarcosAndre28** [reply]
I tried to do it this way but I didn't succeed, I'm still getting the error

**Reply by vshcherb** [reply]
All our builds are passing right now https://builder.osmand.net:8080/view/OsmAnd%20Builds/job/OsmAnd-main-opengl-arm/.  You can search when this class was introduced https://github.com/osmandapp/osmand, https://github.com/osmandapp/osmand-core

**Comment by alex-osm** [comment]
You need to update these libs:
```
	opengldebugImplementation "net.osmand:OsmAndCore_androidNativeDebug:0.1-SNAPSHOT@aar"
	opengldebugImplementation "net.osmand:OsmAndCore_android:0.1-SNAPSHOT@aar"

	openglImplementation "net.osmand:OsmAndCore_androidNativeRelease:0.1-SNAPSHOT@aar"
	openglImplementation "net.osmand:OsmAndCore_android:0.1-SNAPSHOT@aar"
```
https://github.com/osmandapp/OsmAnd/blob/master/OsmAnd/build.gradle#L219

Ie refresh dependencies.

**Reply by MarcosAndre28** [reply]
sorry, how do i update these libraries ??????

**Reply by MarcosAndre28** [reply]
this library is imported from an alternative package ?
